signac/__main__.py

```diff
@@ -279,6 +279,12 @@ def main_job(args):
     if args.create:
         job.init()
     if args.workspace:
+        warnings.warn(
+            "The `-w/--workspace` parameter is deprecated. Use -p/--path instead",
+            FutureWarning,
+        )
```

Comment:
Should we mention "deprecated as of ... and will be removed in ..." and then add the corresponding assertion?

Reply:
Good question. I don't think this change needs a particularly long deprecation, so are we fine with removing this in 2.0?

Reply:
I would be.

Reply:
Yes, I’m also in favor.